repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/user.st

```diff
@@ -0,0 +1,4 @@
+accessing
+user
+
+	^ user ifNil: [ ^ '' ]
```

Comment:
Same here: wouldn't it make things easier to return `user` directly?

Reply:
IIRC for some client is was important not to pass `nil` as a user. Nevertheless, I will take a second look at this when implementing `SCIAbstractLoadSpecWithCredentials` for #486 when I find some time for it :-)